service/README.md

```diff
@@ -11,6 +11,25 @@ Currently, the OpenTelemetry Collector supports the following providers `scheme`
 
 For more technical details about how configuration is resolved you can read the [configuration resolving design](../confmap/README.md#configuration-resolving).
 
+One configuration provider can also make references to other config providers, like the following:
```

Comment:
Should this also be a level 3 header as "single config source". Say something like "embed config" ?

Reply:
I would actually move the env vars part into here, I feel like it fits better here in the general part. I can move it down if most people prefer it that way though.

Reply:
If i'm understanding Bogdan's suggestion correctly, he's suggesting adding a heading here. I support this as it will make linking directly to "embedding configuration" easier.

Reply:
I was about to write exactly this in the morning (forgot to press submit), that make links easier.

Reply:
Sorry, I mixed up the PRs: the website doc has another section towards the end of the page with the docs for "env". I agree with you on this one being part of a new heading.